config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -88,6 +88,11 @@
 #
 # portal.google.universal_analytics_id=UA-XXXXXX-XX
 
+# SAK-41039
+# Support to anonymize IP in Google Analytics
```

Comment:
Worth mentioning that it only works with Universal Google Analytics?

Reply:
Yes I guess, the patch comes from the DEV list and it's inside the universal GA if.